prime-router/metadata/hl7_mapping/STLTs/CA/ca-observation-request.yml

```diff
@@ -0,0 +1,14 @@
+constants:
+  hl7Order: '/PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/OBR'
+  senderOrganization: '%resource.basedOn.resolve().performer.resolve()'
+elements:
+  - name: ca-order-entity-id
+    value: [ '%resource.identifier[0].value' ]
+    hl7Spec: [ '%{hl7Order}-3-1' ]
+
+  - name: ca-order-organization-identifier
+    resource: '%senderOrganization'
+    constants:
+      organizationIdFieldPath: '%{hl7Order}-3'
+    schema: ../../common/organization-identifier
+
```

Comment:
We are not receiving identifiers for Diagnostic Reports from SimpleReport we need to grab the identifier values from the ServiceRequest Organization.